tests/time_tests/test_runner/conftest.py

```diff
@@ -194,6 +195,7 @@ def pytest_runtest_makereport(item, call):
             else:
                 data["status"] = "passed"
         upload_timetest_data(data, db_url, db_collection)
+        logging.info("Upload data to {}/{}.{}. Data: {}".format(db_url, DATABASE, db_collection, data))
```

Comment:
Better to have the message before the action so that we can understand what action is in progress:
```
logging.info("Uploading...
upload_timetest_data(...
```

Reply:
Moved